server/src/main/java/com/epam/aidial/core/server/service/ApplicationService.java

```diff
@@ -401,11 +412,11 @@ public Application undeployApplication(ResourceDescriptor resource) {
         return result.getValue();
     }
 
-    public Application.Logs getApplicationLogs(ResourceDescriptor resource) {
+    public Application.Logs getApplicationLogs(ResourceDescriptor resource, ProxyContext context) {
         verifyApplication(resource);
         controller.verifyActive();
 
-        Application application = getApplication(resource).getValue();
+        Application application = getApplication(resource, context).getValue();
```

Comment:
nit. You do not need to change application here.

Reply:
removed